File: python/class 7/question 6.py
list = [float(input('Enter a number: ')) for i in range(6)]

positive_numbers=[]
negative_numbers=[]

for num in list: 
  if num > 0: 
    positive_numbers.append(list)
  elif num < 0: 
    negative_numbers.append(list)

print(len(positive_numbers), sum(negative_numbers))







#alternatively you could do 
list = [float(input('Enter a number: ')) for i in range(6)]
positive_numbers=[num for num in list if num > 0]
negative_numbers=[num for num in list if num < 0]
print(len(positive_numbers), sum(negative_numbers))

File: java/class 6/Question6.java
import java.util.Scanner;

public class Question6 {
  public static void main(String[] args){
    int positiveNumbers = 0;
    int negativeNumbers = 0;
    int input;
    Scanner s = new Scanner(System.in);
    for(int i = 1; i <= 6; i++){
      input = s.nextInt();
      if (input>0) {
        positiveNumbers++;
      } else if (input<0){
        negativeNumbers+=input;
      }
    }
    System.out.println("The number of positive numbers is " + positiveNumbers + " and the sum of the negative numbers is " + negativeNumbers);
    
    s.close();
  }
}

